translations/el/LC_MESSAGES/messages.po

```diff
@@ -41,20 +42,20 @@ msgstr "Hedy - My programs"
 #: app.py:785 app.py:795 app.py:799 app.py:813 app.py:1050 app.py:1353
 #: website/admin.py:17 website/admin.py:25 website/admin.py:95
 #: website/admin.py:112 website/admin.py:130 website/auth.py:711
-#: website/auth.py:738 website/statistics.py:86
+#: website/auth.py:738 website/statistics.py:86 app.py:1051 app.py:1354
 #, fuzzy
 msgid "unauthorized"
 msgstr "You don't have access rights for this page"
 
-#: app.py:823
```

Comment:
hmmm interesting! It seems the po files store their location in the code, so we will get updates to the po files when changes are made unrelated to the translations?